### ISSUE TITLE
Is it possible to map keys in EDIT MODE?

### ISSUE THREAD

**Issue by SeerLite** [andmarti1424/sc-im#692]
Hi! This is an amazing spreadsheet editor, thanks for making it :)

Is it possible to map keys in EDIT MODE (the mode that one enters when pressing `e`)?

I use a non-standard keyboard layout and remap the Vi directional keys on most programs I use, and I'd like to be able to do that in edit mode too. I couldn't find anything in the `:help` page about this.

Thanks!

**Comment by andmarti1424** [comment]
No. I am afraid mappings are not supported in EDIT_MODE. Patches are always welcome though!
